src/pages/data-reporting/reports-api/v2.0/index.mdx

```diff
@@ -25,6 +25,22 @@ You can access all the request from the Reports API v2 in the Postman Collection
 
 <Section>
 
+## Sample App
```

Comment:
```suggestion
## Sample app
```

Reply:
I was wondering if we could add a smaller image for this app as it's quite large